tests/gene-comparison-tool.spec.ts

```diff
@@ -27,4 +27,46 @@ test.describe('specific viewport block', () => {
     const dropdown = page.locator('#subCategory');
     await expect(dropdown).toHaveText('Targeted Selected Reaction Monitoring (SRM)');
   });
+
+  test('switching from RNA to Protein with RNA-specific column ordering reverts back to Risk Score descending', async ({ page }) => {
+    // set category for Protein - Differential Expression
+    await page.goto('/genes/comparison?sortField=FP&sortOrder=1');
+
+    // wait for page to load (i.e. spinner to disappear)
+    await expect(page.locator('div:nth-child(4) > div > .spinner'))
+      .not.toBeVisible({ timeout: 150000});
```

Comment:
Consider using the newly added helper method here? 

```suggestion
    await waitForSpinnerNotVisible(page, 150000);
```

Reply:
using new helper method